recipes/starshot/recipe.yml

```diff
@@ -1,6 +1,6 @@
-name: Drupal CMS
-description: Applies all the Drupal CMS recipes.
-type: Drupal CMS
+name: Starshot
+description: Applies all the Starshot recipes.
+type: Starshot
```

Comment:
Small point - this should actually be `Site`. :) (It's a quirk of the recipe initative.)